openreview/conference/invitation.py

```diff
@@ -1501,7 +1501,7 @@ def set_paper_ranking_invitation(self, conference, group_id, start_date, due_dat
                     "tag": {
                         "description": "Select value",
                         "order": 1,
-                        "value-regex": 'No Ranking|[0-9]+\sof\s[0-9]+',
+                        "value-regex": r'No Ranking|[0-9]+\sof\s[0-9]+',
```

Comment:
Why are you changing this? does it return an string?

Reply:
That is just a way to indicate to pylint that there are backslash chars in the string.